ts/de_DE.ts

```diff
@@ -52,7 +52,7 @@
     <message>
         <location filename="controller.py" line="1837"/>
         <source>Exporting G-code to</source>
-        <translation>Exportiere GCode nach</translation>
+        <translation>Exportiere gcode nach</translation>
```

Comment:
I would rarther suggest GCode, G-code or G-Code instead of gcode. It's written that way most of the time:
https://en.wikipedia.org/wiki/G-code
http://reprap.org/wiki/G-code
https://thingiverse-production-new.s3.amazonaws.com/assets/87/b0/2c/f5/4c/CheatSheet.pdf